client/src/pages/Resources.jsx

```diff
@@ -164,6 +168,12 @@ function Resources({
     setFilteredResources(
       newResources == null ? [] : newResources.result.totalData,
     );
+    setResourceCount(
+      newResources == null
+        ? []
+        : newResources.result.totalCount[0].resourceCount,
+    );
```

Comment:
Since `resourceCount` is a integer, if `newResources` is `null` maybe we can set it to 0 instead?